src/main/java/com/google/sps/model/Trip.java

```diff
@@ -6,6 +6,7 @@ public class Trip {
     private long tripID;
     private String title;
     private List<Long> participants;
+    private List<Event> itenerary;
```

Comment:
do you think we should store the events in the link or just store their ids, same to the user lists.

Reply:
I think you should store the ids.

Reply:
Got it. Updating the lists to store longs that will be the ids